plugin-azure-server/src/main/kotlin/jetbrains/buildServer/clouds/azure/arm/AzureCloudClient.kt

```diff
@@ -27,14 +27,15 @@ import jetbrains.buildServer.clouds.base.connector.CloudApiConnector
  */
 class AzureCloudClient(private val params: CloudClientParameters,
                        apiConnector: CloudApiConnector<AzureCloudImage, AzureCloudInstance>,
-                       imagesHolder: AzureCloudImagesHolder)
+                       imagesHolder: AzureCloudImagesHolder,
+                       private val myKubePodNameGenerator: KubePodNameGenerator)
```

Comment:
Should this be renamed?

Reply:
I'm trying to keep with the convention of the already written plugin, where the possessive "my" is added for a value/object of the class. I don't think this should be renamed therefore.

Reply:
Sorry, I was unclear there - I was referring to `KubePodNameGenerator`.

Reply:
Oh thats the class object. What names were you thinking instead? Something more azure specific?